global-news-app/src/Utils/index.ts

```diff
@@ -1,5 +1,31 @@
 import { format, parseISO } from "date-fns";
 import { dateFormat } from "./consts";
+import { Selections } from "Types";
+import { SelectLabels, SelectionsKeys } from "Types/enums";
 
-export const formattedDate = (publishedAt: string) =>
-  format(parseISO(publishedAt), dateFormat);
+export const formattedDate = (publishedAt: string) => {
+  return format(parseISO(publishedAt), dateFormat);
+};
+
+export const constructNewsApiUrl = (selections: Selections): string => {
+  const params = new URLSearchParams();
+
+  const { articleType, ...otherSelections } = selections;
+  const formatteAarticleType =
+    articleType === SelectLabels.EVERYTHING
+      ? SelectionsKeys.EVERYTHING
+      : SelectionsKeys.TOP_HEADLINES;
+  const baseUrl = `https://newsapi.org/v2/${formatteAarticleType}?`;
```

Comment:
consider configuring base url with React Query or axios
 

https://stackoverflow.com/questions/72617227/is-there-a-way-to-configure-a-base-url-on-react-querys-queryclient

https://axios-http.com/docs/config_defaults